data/Slic3r Prusa Edition

```diff
@@ -1 +1,2 @@
-https://github.com/prusa3d/Slic3r/releases
+https://github.com/prusa3d/Slic3r/releases/download/version_1.41.2/Slic3rPE-1.41.2+linux64-full-201811221508.AppImage
+# Link without version would be better
```

Comment:
Well then my link would be better right ?
Yours HAS a version....

Reply:
Yes, but `worker.sh` is not smart enough to handle it.